apps/laboratory/src/hooks/useWalletGetAssets.ts

```diff
@@ -35,12 +35,12 @@ async function getAssetDiscoveryCapabilities({
   try {
     // For WalletConnect, also check CAIP-25
     if (walletProviderType === 'WALLET_CONNECT') {
-      const scopedProperties = JSON.parse(
-        //@ts-expect-error - currently scopedProperties is not types
-        provider.session?.scopedProperties || '{}'
-      )
-      const eip155Capabilities = scopedProperties?.eip155
-      const walletService = eip155Capabilities?.walletService
+      const eip155Capabilities =
+        typeof provider.session?.scopedProperties?.['eip155'] === 'string'
+          ? JSON.parse(provider.session.scopedProperties['eip155'])
+          : {}
+
+      const walletService = eip155Capabilities?.walletService || []
```

Comment:
Let's use constant from `@reown/appkit-common`

```js
import { ConstantsUtil } from '@reown/appkit-common'
```

```suggestion
      const eip155Capabilities =
        typeof provider.session?.scopedProperties?.[ConstantsUtil.CHAIN.EVM] === 'string'
          ? JSON.parse(provider.session.scopedProperties[ConstantsUtil.CHAIN.EVM])
          : {}

      const walletService = eip155Capabilities?.walletService || []
```

Reply:
updated